src/Microsoft.AspNetCore.Blazor.Browser/Interop/InvocationResult.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Blazor.Browser.Interop
+{
+    internal class InvocationResult<TRes>
+    {
+        // Whether the method call succeeded or threw an exception.
+        public bool Succeeded { get; set; }
```

Comment:
Really minor but it looks like all the setters in this type can be `private`

Reply:
No forget that, I guess they have to be public for JSON deserialization.

Will delete these two comments.